akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowConflateSpec.scala

```diff
@@ -168,6 +169,9 @@ class FlowConflateSpec extends AkkaSpec {
       sub.expectRequest(1)
       sub.sendNext(3)
 
+      // delay requesting to avoid race condition
```

Comment:
The race is between the sendNext above and the request below (even though they come next to each other in the code) due to the interpreter in fuzzing mode reordering events that otherwise would be sequential due to the enclosing actor's mailbox.